src/server/src/projects/entities/create-project.dto.ts

```diff
@@ -18,6 +27,10 @@ export class CreateProjectDto implements CreateProjectData {
   @IsOptional()
   readonly districtsDefinition: DistrictsDefinition;
   @IsOptional()
+  @IsNumber()
+  @Max(100, { message: "Population deviation must be between 0% and 100%" })
```

Comment:
We don't want to allow a negative deviation.
```suggestion
  @Max(100, { message: "Population deviation must be between 0% and 100%" })
  @Min(0, { message: "Population deviation must be between 0% and 100%" })
```